agent/ansible/pbench/agent/roles/epel_repo_install/README.md

```diff
@@ -0,0 +1,34 @@
+epel_repo_install
+=================
+
+This role installs a repo file for EPEL.
+
+Requirements
+------------
+
+Role Variables
+--------------
+
+Dependencies
+------------
+
+Example Playbook
+----------------
+An example playbook can be obtained from
+
+    https://github.com/distributed-system-analysis/pbench/blob/master/agent/ansible/pbench_agent_install.yml
+
+or you can use the raw link to wget/curl the file:
+
+    https://raw.githubusercontent.com/distributed-system-analysis/pbench/master/agent/ansible/pbench_agent_install.yml
```

Comment:
That's interesting: I've never seen this `raw.githubusercontent.com` path. What's the difference, and why is it worth including both?

Reply:
It's where you go if you click on the "Raw" link of a file on Github. CLI addicts tend to prefer that kind of link, so they don't have to deal with HTML junk.